OrbitCore/LibunwindstackUnwinder.cpp

```diff
@@ -0,0 +1,68 @@
+#include "LibunwindstackUnwinder.h"
+
+#include "PrintVar.h"
+
+bool LibunwindstackUnwinder::SetMaps(const std::string& maps_buffer) {
+  maps_ = std::make_unique<unwindstack::BufferMaps>(maps_buffer.c_str());
+
+  if (!maps_->Parse()) {
+    PRINT("LibunwindstackUnwinder::SetMaps: !maps_->Parse()\n");
```

Comment:
I would write a more descriptive error message here. And don't we have something like PRINT_ERR?

Reply:
PRINT is what is used for errors in the rest of the codebase. I changed to message.